examples/sw/simple_system/common/crt0.S

```diff
@@ -46,6 +46,8 @@ reset_handler:
   mv x30, x1
   mv x31, x1
 
+  csrr x1, mstatus
```

Comment:
No big deal, but could you write something in the commit message that explains why this change is in this commit?